test/test-platform/TestPlatform.cpp

```diff
@@ -333,14 +333,6 @@ bool CTestPlatform::createInclusiveSelectionCriterionFromStateList(
     assert(pCriterionType != NULL);
 
     uint32_t uiNbStates = remoteCommand.getArgumentCount() - 1;
-
-    if (uiNbStates > 32) {
-
-        strResult = "Maximum number of states for inclusive criterion is 32";
-
-        return false;
-    }
-
```

Comment:
This mean that `0x1 << uiState` might overflow => UB ! Maybe we could replace 0x1 => 1U.